app/tests/spec/views/sign_in_totp_code.js

```diff
@@ -67,6 +67,9 @@ describe('views/sign_in_totp_code', () => {
 
     sinon.stub(view, 'getSignedInAccount').callsFake(() => model.get('account'));
 
+    $('body').attr('data-flow-id', 'F1031DF1031DF1031DF1031DF1031DF1031DF1031DF1031DF1031DF1031DF103');
+    $('body').attr('data-flow-begin', '42');
+
```

Comment:
Maybe further down, in the `validateAndSubmit` block, we could also stub `logViewEvent` and make sure it's called for the `success` case?